vnext/ReactUWP/Views/cppwinrt/react.uwp.image.ReactImageBrush.g.h

```diff
@@ -0,0 +1,83 @@
+﻿// WARNING: Please don't edit this file. It was generated by C++/WinRT v1.0.180821.2
+
+#pragma once
+
+#include "winrt/Windows.Foundation.h"
+#include "winrt/Windows.UI.Composition.h"
+#include "winrt/Windows.UI.Xaml.h"
+#include "winrt/Windows.UI.Xaml.Media.h"
+#include "winrt/react.uwp.image.h"
+
+namespace winrt::react::uwp::image::implementation {
+
+template <typename D, typename... I>
+struct WINRT_EBO ReactImageBrush_base : implements<D, react::uwp::image::IReactImageBrush, Windows::UI::Xaml::Media::IBrushOverrides2, Windows::UI::Xaml::Media::IXamlCompositionBrushBaseOverrides, composing, I...>,
+    impl::require<D, Windows::UI::Composition::IAnimationObject, Windows::UI::Xaml::IDependencyObject, Windows::UI::Xaml::IDependencyObject2, Windows::UI::Xaml::Media::IBrush, Windows::UI::Xaml::Media::IXamlCompositionBrushBase, Windows::UI::Xaml::Media::IXamlCompositionBrushBaseProtected>,
+    impl::base<D, Windows::UI::Xaml::Media::XamlCompositionBrushBase, Windows::UI::Xaml::Media::Brush, Windows::UI::Xaml::DependencyObject>,
+    Windows::UI::Xaml::Media::IBrushOverrides2T<D>, Windows::UI::Xaml::Media::IXamlCompositionBrushBaseOverridesT<D>
+{
+    using base_type = ReactImageBrush_base;
+    using class_type = react::uwp::image::ReactImageBrush;
+    using implements_type = typename ReactImageBrush_base::implements_type;
+    using implements_type::implements_type;
+    using composable_base = Windows::UI::Xaml::Media::XamlCompositionBrushBase;
+#if _MSC_VER < 1914
+    operator class_type() const noexcept
+    {
+        static_assert(std::is_same_v<typename impl::implements_default_interface<D>::type, default_interface<class_type>>);
+        class_type result{ nullptr };
+        attach_abi(result, detach_abi(static_cast<default_interface<class_type>>(*this)));
+        return result;
+    }
+#else
+    operator impl::producer_ref<class_type> const() const noexcept
+    {
+        return { to_abi<default_interface<class_type>>(this) };
+    }
+#endif
+
+    hstring GetRuntimeClassName() const
+    {
+        return L"react.uwp.image.ReactImageBrush";
+    }
+    ReactImageBrush_base()
+    {
+        impl::call_factory<Windows::UI::Xaml::Media::XamlCompositionBrushBase, Windows::UI::Xaml::Media::IXamlCompositionBrushBaseFactory>([&](auto&& f) { f.CreateInstance(*this, this->m_inner); });
+    }
+};
+
+}
+
+namespace winrt::react::uwp::image::factory_implementation {
+
+template <typename D, typename T, typename... I>
+struct WINRT_EBO ReactImageBrushT : implements<D, Windows::Foundation::IActivationFactory, I...>
```

Comment:
>ReactImageBrushT [](start = 17, length = 16)

Can you add the .idl file you used to create the generated files somewhere in the repo?  I don't see any additions to say ReactUWP\Views\cppwinrt\reactuwp.idl  where the existing one lives.  So when we fix things to generate in the build ...